pt/convolutional-neural-networks.md

```diff
@@ -0,0 +1,718 @@
+
+<br>
+
+
+**21. [Preserves detected features, Most commonly used, Downsamples feature map, Used in LeNet]**
+
+&#10230; [Preserva os recursos detectados, Mais comumente usados, Mapa de recursos de amostragem, Usado no LeNet]
+
+
+<br>
+
+
+**22. Fully Connected (FC) ― The fully connected layer (FC) operates on a flattened input where each input is connected to all neurons. If present, FC layers are usually found towards the end of CNN architectures and can be used to optimize objectives such as class scores.**
+
+&#10230; Totalmente Conectado (FC) - A camada totalmente conectada (FC opera em uma entrada achatada, onde cada entrada é conectada a todos os neurônios. Se estiver presente, as camadas FC geralmente são encontradas no final das arquiteturas da CNN e podem ser usadas para otimizar objetivos, como pontuações de classes.
+
+<br>
+
+
+**23. Filter hyperparameters**
+
+&#10230; Hiperparâmetros de filtros
+
+<br>
+
+
+**24. The convolution layer contains filters for which it is important to know the meaning behind its hyperparameters.**
+
+&#10230; A camada de convolução contém filtros para os quais é importante conhecer o significado por trás de seus hiperparâmetros.
+
+<br>
+
+
+**25. Dimensions of a filter ― A filter of size F×F applied to an input containing C channels is a F×F×C volume that performs convolutions on an input of size I×I×C and produces an output feature map (also called activation map) of size O×O×1.**
+
+&#10230; Dimensões de um filtro - Um filtro de tamanho F×F aplicado a uma entrada contendo C canais é um volume de tamanho F×F×C que executa convoluções em uma entrada de tamanho I×I×C e produz um mapa de recursos (também chamado de mapa de ativação) da saída de tamanho O×O×1.
+
+<br>
+
+
+**26. Filter**
+
+&#10230; Filtros
+
+<br>
+
+
+**27. Remark: the application of K filters of size F×F results in an output feature map of size O×O×K.**
+
+&#10230; Observação: a aplicação de K filtros de tamanho F×F resulta em um mapa de recursos de saída de tamanho O×O×K.
+
+<br>
+
+
+**28. Stride ― For a convolutional or a pooling operation, the stride S denotes the number of pixels by which the window moves after each operation.**
+
+&#10230; Passo - Para uma operação convolucional ou de pooling, o passo S denota o número de pixels que a janela se move após cada operação.
+
+<br>
+
+
+**29. Zero-padding ― Zero-padding denotes the process of adding P zeroes to each side of the boundaries of the input. This value can either be manually specified or automatically set through one of the three modes detailed below:**
+
+&#10230; Zero preenchimento (Zero-padding) - Zero preenchimento denota o processo de adicionar P zeros em cada lado das fronteiras do input. Esse valor pode ser especificado manualmente ou automaticamente ajustado através de um dos três modelos abaixo:
```

Comment:
Don't you think is better to translate _input_ and use _entrada_?

Reply:
totally! my mistake